lib/cli.js

```diff
@@ -44,7 +44,8 @@ const SUPPORTED_SERVICES = [
     ' phantomjs - https://github.com/cognitom/wdio-phantomjs-service',
     ' static-server - https://github.com/LeadPages/wdio-static-server-service',
     ' visual-regression - https://github.com/zinserjan/wdio-visual-regression-service',
-    ' webpack - https://github.com/leadpages/wdio-webpack-service'
+    ' webpack - https://github.com/leadpages/wdio-webpack-service',
+    ' webpack-dev-server - https://gitlab.com/Vinnl/wdio-webpack-dev-server-service'
```

Comment:
Hmm, I must admit that I hadn't even thought of that - mostly because Webpack and Webpack Dev Server are (theoretically) independent projects too, I guess. I'm not quite sure exactly what the use case of the Webpack service is -I'm guessing it should perhaps be used in combination with the static file server?-, but I guess we could add an option `webpackServeAt` or something that, if configured, has the effect of starting a server at the given port rather than bundling the assets? My main reservation would be that it might not be that clear when to use which combination of configuration options, since the two use cases appear to be pretty different. (i.e. `webpack` for testing a production-like build, `webpack-dev-server` for locally running code.)

What do you think?